packages/common-all/src/StatisticsUtils.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Utility class for _very simple_ statistics operations.
+ * When we need to do complex and/or expensive stats,
+ * consider introducing a dedicated stats library.
+ *
+ * This is here to use until that point comes
+ */
+
+import _ from "lodash";
+import { DendronError } from ".";
+
+export class StatisticsUtils {
+  /**
+   * Get standard deviation from array of numbers.
+   */
+  static stddev(arr: number[]) {
```

Comment:
since this will be used in many places and its not obvious an error could be thrown, can we return a `RespV3` so that this is explicit?

Reply:
went with disallowing empty arrays altogether by narrowing the argument type.